Firebase/InstanceID/Private/FIRInstanceID_Private.h

```diff
@@ -41,13 +45,6 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable NSString *)token;
 
-/**
- *  Verify if valid checkin preferences have been loaded in memory.
- *
- *  @return YES if valid checkin preferences exist in memory else NO.
- */
-- (BOOL)hasValidCheckinInfo;
```

Comment:
Was this intentional?

Reply:
Yep, so Travis CI failed because XCode generated a warning where FIRInstanceID does not have an implementation for hasValidCheckinInfo. I think this is old code since there's no references that method in FIRInstanceID. It looks like the code got refactored and that method no lives in FIRInstanceIDCheckinPreferences.